snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -97,10 +97,19 @@ public class Snapshot: NSObject {
         print("snapshot: \(name)") // more information about this, check out https://github.com/fastlane/fastlane/tree/master/snapshot
 
         sleep(1) // Waiting for the animation to be finished (kind of)
-        XCUIDevice.sharedDevice().orientation = .Unknown
+
+        #if os(tvOS)
+            XCUIApplication().childrenMatchingType(.Browser).count
```

Comment:
I tried it on iOS and it works too, as expected. Could be refactored to remove the Orientation change, but I'll leave that for a separate PR. Also the "Browser" check is probably safer on tvOS only since there are no web-browsers there.